src/contact.js

```diff
@@ -14,6 +14,7 @@ class Contact {
     this.id = o.id;
     this.mdid = o.mdid;
     this.name = o.name;
+    this.senderName = o.senderName;
```

Comment:
Do we really need that? I think android is just asking for the name you want to share every time you invite somebody. The less extra information we have on the facilitated tx list the better